socketcan_interface/socketcan_interface-extras.cmake

```diff
@@ -0,0 +1,7 @@
+find_package(Boost REQUIRED
+  COMPONENTS
+    chrono
+    system
+    thread
+)
+list(APPEND pluginlib_LIBRARIES ${Boost_LIBRARIES})
```

Comment:
Why do you append to `pluginlib_LIBRARIES`?

Reply:
Whoops, that is left over from the code where I based this PR on. I'll clean it up